update.sh

```diff
@@ -167,6 +167,15 @@ function update_node_version() {
       )\""
       sed -Ei -e "s/(alpine:)0.0/\\1${alpine_version}/" "${dockerfile}-tmp"
       sed -Ei -e "s/CHECKSUM=CHECKSUM_x64/CHECKSUM=${checksum}/" "${dockerfile}-tmp"
+
+      # Replace python3 with python2 for nodejs < 14 on alpine
+      if [ "$version" -lt 14 ]; then
+        pythonVersion="python2"
```

Comment:
We could probably keep this as the old "python" and it would minimize the diff

Reply:
I'll have to add some additional logic since "python" isn't available on alpine3.12, and there there is 10/alpine3.12 and 12/alpine3.12

Reply:
OK, fair enough, was just looking to see what the minimal change would be. If this is it, that should be good

Reply:
I could add a check for alpine < 3.12 and nodejs < 14 and have it use "python" in that case. Would that be preferred?

Reply:
I think it looks good now, thanks for the patients